src/hooks/useEditorView.ts

```diff
@@ -137,5 +146,13 @@ export function useEditorView<T extends HTMLElement = HTMLElement>(
     if (stateProp) view?.setProps({ state: stateProp });
   }, [view, stateProp]);
 
+  useLayoutEffect(() => {
+    editorPropsRef.current = withConditionalFlushDispatch(
+      props,
+      forceUpdate,
+      view
+    );
+  }, [props, view]);
```

Comment:
Looks like we're missing forceUpdate here. It has a stable identity, so it's safe to add to the dependencies

Reply:
aah, needed to restart my eslint server on vscode, pushing up